google/oauth2/credentials.py

```diff
@@ -79,6 +82,7 @@ def __init__(
         rapt_token=None,
         refresh_handler=None,
         enable_reauth_refresh=False,
+        granted_scopes=None,
```

Comment:
since granted scopes are not given by the user, I guess we should remove it from the constructor? Inside the class we still keep the _granted_scopes and let the user use the property to access the value.